src/System.Runtime/tests/System/GuidTests.cs

```diff
@@ -346,21 +370,46 @@ public static void ToByteArray()
             Assert.Equal(new byte[] { 0xd5, 0x10, 0xa1, 0xa8, 0x49, 0xfc, 0xc5, 0x43, 0xbf, 0x46, 0x80, 0x2d, 0xb8, 0xf8, 0x43, 0xff }, s_testGuid.ToByteArray());
         }
 
+        [Theory]
+        [MemberData(nameof(Ctor_ByteArray_TestData))]
+        public static void TryWriteBytes(byte[] b, Guid destination)
+        {
+            byte[] bytes = new byte[16];
+            destination.TryWriteBytes(new Span<byte>(bytes));
+            Assert.Equal(b, bytes);
+        }
+
+        [Theory]
+        [MemberData(nameof(Ctor_ByteArray_TestData))]
+        public static void TryWriteBytes_ReturnsTrue(byte[] b, Guid destination)
```

Comment:
Why `destination` ... isn't it the input?

Reply:
You're right, should be the input! Got the naming mixed up with the span passed in as a destination.